cli/CHANGELOG.md

```diff
@@ -1,17 +1,22 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 12.10.1
+## 12.11.0
 
-_Released 04/25/2023 (PENDING)_
+_Released 04/25/2023_
 
 **Features:**
-
+- Adds Component Testing support for Angular 16. Addresses [#26044](https://github.com/cypress-io/cypress/issues/26044).
```

Comment:
There needs to be an empty line break after each heading

Reply:
Added in 9a86afe7b40d7d013d969923d03917afac715504